openprescribing/frontend/views/views.py

```diff
@@ -1364,8 +1356,10 @@ def _handle_bookmark_and_newsletter_post(
             request.session['alerts_requested'] = 1
             user = _authenticate_possibly_new_user(email)
             form_args = _make_bookmark_args(user, form, subject_field_ids)
-            form_args['approved'] = _unapprove_bookmarks_when_different_user(
-                user, request)
+            _unverify_email_address_when_different_user(user, request)
+            # We're automatically approving all alert signups from now on
+            # without waiting for the email address to be verified
+            form_args['approved'] = True
```

Comment:
Do we need `approved` on the form anymore?

Reply:
I don't think we can (yet) get rid of the `approved` state because we don't want to retroactively approve all the existing bookmarks and start sending emails to people who have never got them before.

We talked about maybe doing a mailout to all address who never verified and ask if they'd like to start receiving them.

Reply:
I see.